pkg/loader/fileloader.go

```diff
@@ -160,33 +161,45 @@ func (l *fileLoader) New(path string) (ifc.Loader, error) {
 	if path == "" {
 		return nil, fmt.Errorf("new root cannot be empty")
 	}
-	if git.IsRepoUrl(path) {
-		// Avoid cycles.
-		if err := l.errIfPreviouslySeenUri(path); err != nil {
+	repoSpec, err := git.NewRepoSpecFromUrl(path)
```

Comment:
`NewRepoSpecFromUrl` is intended to replace `IsRepoUrl`. Maybe we can replace all the usage of `IsRepoUrl` and replace them in a subsequent PR.

Reply:
yep, i plan to remove it.